data/question-data.json

```diff
@@ -1,3 +1,9 @@
 {
-    "title": "The Test Question"
+    "title": "The Test Question",
+    "options": {
+        "A": "Answer A",
+        "B": "Answer B",
+        "C": "Answer C"
+    },
```

Comment:
I think the `options` would be better represented as an array of objects. A `correct` flag should be attached to the correct `option`:
`
"options" : [
  {"text" : "example answer"},
  {"text" : "other answer"},
  {"text": "the correct answer", "correct": true}
] 
`
This will allow rearranging options easier (maybe randomised) and not tie them to a specific identifer (ABC or 123).